src/pathOrLazy.js

```diff
@@ -0,0 +1,30 @@
+import { curryN, identical, partial, pathOr, unary, when } from 'ramda';
+
+/**
+ * If the given, non-null object has a value at the given path, returns the value at that path.
+ * Otherwise returns the result of invoking the provided function with the object.
+ *
+ * @func pathOrLazy
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.22.0|v2.22.0}
+ * @category Object
+ * @typedefn Idx = String | Int
+ * @sig (() -> a) -> [Idx] -> {a} -> a
```

Comment:
`@sig ({a} -> a) -> [Idx] -> {a} -> a` - this should be more correct reflecting on the fact we provide the original object back to defaultFn